Makefile

```diff
@@ -14,23 +16,21 @@ QEMU := qemu-system-aarch64
 .PHONY: all
 all: $(IMG)
 
-$(EXE): linker.ld swift
-	$(LD) $(LDFLAGS) -T linker.ld -Xlinker -Map=$(MAP) .build/release/libKernel.a -o $@
+$(EXE): Makefile $(LINKER_SCRIPT) $(LIB)
+	$(LD) $(LDFLAGS) -T $(LINKER_SCRIPT) -Xlinker -Map=$(MAP) $(LIB) -o $@
 
-$(IMG): $(EXE)
-	$(OBJCOPY) $< -O binary $@
+$(IMG): Makefile $(EXE)
+	$(OBJCOPY) $(EXE) -O binary $@
 
-.PHONY: swift
-swift:
+$(LIB): Makefile .swift-version Package.swift $(wildcard Package.resolved) Sources
```

Comment:
`Sources` doesn't seem to work. Even if I change `Sources/AsmSupport/aarch64/boot.S`, `make all` won't invoke `swift build`.